### ISSUE TITLE
Number pad doesn't work in Commando scope selection

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#3047]
### Short Summary

eg. Using the number pad just enters a 5, it won't select the Packages scope.

You have to use the numbers at the top of the keyboard.

**Comment by Defman21** [comment]
Keycodes.. 😕

**Comment by Naatan** [comment]
How do they work? Nobody knows! Can't explain that.

**Comment by cgchoffman** [comment]
yep, likely @defman21.  @Naatan, it's impossible to know.  We should punt this to defman.

**Comment by Defman21** [comment]
Yeah also you should assign me to the issue because noone except for me knows what keycodes are.
